scripts/conf.cfg

```diff
@@ -0,0 +1,9 @@
+ROOT=/hdd/projekty/fobnail/script-test
+ATTESTER=/hdd/projekty/fobnail/fobnail-attester/bin/fobnail-attester
```

Comment:
These paths are heavily personalized. Maybe we should use `/usr/share/fobnail` by default or something like that.

Reply:
I've changed to standard paths, some variables are no longer needed so I removed them